bindings/wasm/Cargo.toml

```
@@ -37,6 +36,7 @@ wasm-bindgen-test = { version = "0.3" }
 [target.'cfg(target_arch = "wasm32")'.dependencies]
 chrono = { version = "0.4", features = ["wasmbind"] }
 getrandom = { version = "0.2", features = ["js"] }
+parking_lot = { version = "0.11.2", features = ["wasm-bindgen"] }
```

nit: no need to specify the patch, will be overridden anyway. I wonder if these versions can be specified as `"*"` with `default-features = false`, similar to the former tokio workaround?

```suggestion
parking_lot = { version = "0.11", features = ["wasm-bindgen"] }
```